### PR TITLE
Update changelog and docs for v0.17.2

### PR DESCRIPTION
**Pull request by jiayiwang7**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/whatsnew/changelog.md [hunk 1]
```diff
@@ -15,6 +15,20 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
...
+|              | vSphere | Bare Metal |  Nutanix | CloudStack | Snow  |
+|    :---:     |  :---:  |   :---:   |   :---:  |    :---:   | :---: |
+| Ubuntu       | 20.04	 | 20.04     |	20.04             | Not supported	     | 20.04 |  
+|              | 22.04   | 22.04     |  22.04             | Not supported      | Not supported   |
+| Bottlerocket | 1.13.1  | 1.13.1    |  Not supported     | Not supported	     | Not supported   |
+| RHEL         | 8.7	   | 8.7	     |  Not supported     | 8.7	               | Not supported   |
+
+
+### Fixed
+- TODO
```
Comment (chrisdoherty4) [inline docs/content/en/docs/whatsnew/changelog.md:30]:
```suggestion
- Bare Metal: Ensure new worker node groups can reference new machine configs ([#6615](https://github.com/aws/eks-anywhere/issues/6615))
- Bare Metal: Patch `writefile` action to ensure Bottlerocket configs write content or error ([#2441](https://github.com/aws/eks-anywhere-build-tooling/pull/2441))
```

Comment (abhinavmpandey08) [inline docs/content/en/docs/whatsnew/changelog.md:30]: Added support for configuring healthchecks on `EtcdadmCluster`s using `etcdcluster.cluster.x-k8s.io/healthcheck-retries` annotation [aws/etcdadm-controller#44](https://github.com/aws/etcdadm-controller/pull/44).

Comment (ahreehong) [inline docs/content/en/docs/whatsnew/changelog.md:30]: Add check for making sure quorum is maintained before deleting etcd machines https://github.com/aws/etcdadm-controller/pull/46

### Changed
Only delete one etcd machine at a time when multiple are failing healthchecks https://github.com/aws/etcdadm-controller/pull/46

**Review by abhinavmpandey08 — APPROVED** [review]
/lgtm

**Pull request merged**